src/data/seq/seq.lean

```diff
@@ -288,6 +289,22 @@ begin
   rw [h1, h2], apply H
 end
 
+/-- Embed a list as a sequence -/
+def of_list (l : list α) : seq α :=
+⟨list.nth l, λ n h, begin
```

Comment:
This proof can be further simplified if we import `data.list.basic` (see `list.nth_eq_none_iff`). I don't know if it is a good idea or not.

Reply:
I think the import is fine, given the similarities between the `seq` and `list` APIs.